### PR TITLE
Add return-path option to configuration.

### PR DESCRIPTION
**Pull request by maca134**
Package will add a different return path if it is set in the configuration.

### CODE REVIEW

**Comment by frankdejonge** [comment]
Nice option, but can you fix the indenting and add a docs update for it as well? Both are mandatory for new features/options.

**Comment by maca134** [comment]
Everything sorted, doc pull request: https://github.com/fuel/docs/pull/331

File: classes/email/driver.php [hunk 1]
```diff
@@ -651,8 +663,12 @@ public function send($validate = null)
 		$this->set_header('Date', date('r'));
 
 		// Set return path
-		$this->set_header('Return-Path', $this->config['from']['email']);
-
+		if ($this->config['return_path'] !== false) {
+                    $this->set_header('Return-Path', $this->config['return_path']);
```
Comment (frankdejonge) [inline classes/email/driver.php]: still no good. You use tabs right?

**Comment by maca134** [comment]
Done, hopefully

**Comment by frankdejonge** [comment]
Still didn't comply to the coding standards, but fixed that for you. Please do check out the style guide for further contributions, thanks!

**Pull request merged**